var/spack/repos/builtin/packages/py-pyside/package.py

```diff
@@ -42,6 +42,17 @@ def patch(self):
         rpath.append(os.path.join(
             self.prefix, pypkg.site_packages_dir, 'PySide'))
 
+        # Fix subprocess.mswindows check for Python 3.5
+        # https://github.com/pyside/pyside-setup/pull/55
+        filter_file(
+            r'^if subprocess.mswindows:',
+            'mswindows = (sys.platform == "win32")\r\nif mswindows:',
+            "popenasync.py")
+        filter_file(
+            r'subprocess.mswindows:',
```

Comment:
I would try to match the full line just to make it more specific.

Reply:
> I would try to match the full line just to make it more specific.

I'll make this change and re-test.

Reply:
Fixed in aee4538.